packages/app/src/systems/Chains/eth/components/EthAssetCard.tsx

```diff
@@ -0,0 +1,51 @@
+import { cssObj } from '@fuel-ui/css';
+import { Box, CardList, IconButton } from '@fuel-ui/react';
+import type { ReactNode } from 'react';
+
+type EthAssetCardProps = {
+  icon: ReactNode;
+  name: string;
+  onAdd?: () => void;
+  onPress?: () => void;
+  removeIconButton?: ReactNode;
```

Comment:
this should be `onRemove`, icon should be created inside of the component instead of receiving from parent